CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## [Unreleased]
 
+## [0.6.0] - 2025-05-5
+
+- Added alias substitution and use command for streamlined context interactions
+- Introduced event-triggered command execution with context watch
+- Enabled forced alias creation and validation for safer configuration
+- Improved Dockerfile, login popup, and admin dashboard experience
+- Optimized CI/CD, e2e test reliability, and removed redundant config fields
```

Comment:
let's credit the external authors, where appropriate

Reply:
@miraclx Updated both PR's and externals